### PR TITLE
GH-25 - Review Tool Docs

### LINKED ISSUE #25: Add docs for conversion tool
+ conversion tool "docs" link should then point to new section on docs site

### PR DESCRIPTION
**Pull request by fforbeck**
Closes #25

### CODE REVIEW

File: docs/review-tool/README.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
+# Overview
+
+Creating and publishing high-quality "smart" legal agreements from scratch is a though task,
```
Comment (outkaj) [inline docs/review-tool/README.md]: -> tough

File: docs/review-tool/README.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+but now that you know more about OpenLaw Markup Language you can help us to automate this process.
+
+We’re converting tens of thousands of traditional “legacy” agreements, into “blockchain compatible” contracts -
```
Comment (outkaj) [inline docs/review-tool/README.md]: you don't need the comma after agreements

File: docs/review-tool/README.md [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+which are markdown files with our markup language.
+
+The goal is the build a huge library of "smart" legal agreements, so you can use any of these base documents to
```
Comment (outkaj) [inline docs/review-tool/README.md]: -> to build

File: docs/review-tool/README.md [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+
+## Sign In
+
+- [Login](https://conversion.openlaw.io/login)
+
+## Review
+
+### Modes
+
+The Review Tool allows you to whether reviewing a particular agreement -
```
Comment (outkaj) [inline docs/review-tool/README.md]: I think maybe something is missing from this sentence (decide whether to review a particular agreement or start from where you left off?)

File: docs/review-tool/README.md [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+- One agreement can be reviewed by only one user at time.
+- The users will always see the latest changes based on previous reviews.
+- The maximum time to review an agreement is 30 minutes to prevent users from locking the file forever.
+- If you spent more than 30 minutes you will need to start it again by going to the main page and selecting
+  the agreement to review it again.
+  :::
+
+### Entity Classification
+
+The Review Tool identifies different types of entities and convert them into markdown variables. The first
```
Comment (outkaj) [inline docs/review-tool/README.md]: -> converts

File: docs/review-tool/README.md [hunk 6]
```diff
@@ -0,0 +1,243 @@
...
+version detects Organization and Person names. However we often see invalid or partial matches, and that's
```
Comment (outkaj) [inline docs/review-tool/README.md]: -> comma after However

File: docs/review-tool/README.md [hunk 7]
```diff
@@ -0,0 +1,243 @@
...
+why we need to review this information.
+
+The picture bellow shows a partial match of an Organization name, the same might happen for Person names:
```
Comment (outkaj) [inline docs/review-tool/README.md]: -> below
-> split up into two sentences around the comma

File: docs/review-tool/README.md [hunk 8]
```diff
@@ -0,0 +1,243 @@
...
+The final result should look like this:
+
+<center>
+    <img src="./img/review-popup-extended-selection.png" alt="Extented Selection" />
+</center>
+
+If you want to reset to the initial highlight just click on the red `X`, it will undo the selection.
+
+Moving forward to the second case, sometimes the highlighted text is completely wrong, which means - it is not
+an Organization or a Person name. See bellow:
```
Comment (outkaj) [inline docs/review-tool/README.md]: -> below

File: docs/review-tool/README.md [hunk 9]
```diff
@@ -0,0 +1,243 @@
...
+    <img src="./img/review-popup-invalid-match.png" alt="Invalid Match" />
+</center>
+
+`rentable` is not an Organization or a Person, so click on **No** and set the entity type to `None`.
+
+<center>
+    <img src="./img/review-popup-entity-none.png" alt="Entity None" />
+</center>
+
+It will indicate that you have found an invalid match its type is not relevant.
```
Comment (outkaj) [inline docs/review-tool/README.md]: -> match. Its

File: docs/review-tool/README.md [hunk 10]
```diff
@@ -0,0 +1,243 @@
...
+    <img src="./img/review-popup-invalid-entity.png" alt="Wrong Entity" />
+</center>
+
+Clearly `David V. Singer` is not an Organization name. Fix the entity type by clicking on button **No**:
+
+<center>
+    <img src="./img/review-popup-fix-entity.png" alt="Fix Entity" />
+</center>
+
+And set correct entity type (`Person`), it will be applied to the text and move to the next item.
```
Comment (outkaj) [inline docs/review-tool/README.md]: -> you can move to

File: docs/review-tool/README.md [hunk 11]
```diff
@@ -0,0 +1,243 @@
...
+
+At this of this review step all classified entities will be converted into OpenLaw Markdown variables and applied to
```
Comment (outkaj) [inline docs/review-tool/README.md]: -> stage of this review

File: docs/review-tool/README.md [hunk 12]
```diff
@@ -0,0 +1,243 @@
...
+---
+
+### Variable Renaming
+
+After reviewing the entity matches and types it is time to give a proper name to the variables.
+Remember that each entity you have classified in the previous step is converted into a markup variable.
+Some of the variables we already converted for you, such as dates and addresses. That's why do not need to classify
+these two types.
+
+There are several types of variables to rename, currently these are the types that are available:
```
Comment (outkaj) [inline docs/review-tool/README.md]: -> rename. Currently...

File: docs/review-tool/README.md [hunk 13]
```diff
@@ -0,0 +1,243 @@
...
+    <img src="./img/title-suggestion-popup.png" alt="Title Suggestion" />
+</center>
+
+Hit **Save** and all the changes you have suggested will be applied to the editor.
+
+---
+
+### Editor
+
+The text editor allows add/fix variables that were not identified during Entity Classification
```
Comment (outkaj) [inline docs/review-tool/README.md]: -> the option of adding/fixing

File: docs/review-tool/README.md [hunk 14]
```diff
@@ -0,0 +1,243 @@
...
+and Variable Renaming steps.
+
+<center>
+    <img src="./img/editor.png" alt="Editor" />
+</center>
+
+You can add new variables, remove lines and fix variables names in the text, after you finish your
```
Comment (outkaj) [inline docs/review-tool/README.md]: -> text. After...

File: docs/review-tool/README.md [hunk 15]
```diff
@@ -0,0 +1,243 @@
...
+
+If you have to add anything else after you have submitted, go back to home page and select the exact same
+agreement to start a new review round. The new review round will start with your latest changes, so you
+can edit the file and upload it again.
+
+---
+
+### Feedback
+
+For each review uploaded review the tool asks some feedback about the process. It is completely optional
```
Comment (outkaj) [inline docs/review-tool/README.md]: I think review doesn't need to be twice here?

**Comment by fforbeck** [comment]
@outkaj I've made the changes you suggested but when I rebased my branch on master it broke the `npm test`. So I executed the `npm run format` and pushed the changes to fix that.

**Comment by outkaj** [comment]
Thanks for fixing the test. Approving and merging.

**Pull request merged**